.gitignore

```diff
@@ -353,4 +353,14 @@ MigrationBackup/
 /build
 /mimxrt1060
 /build_linux
+/build_windows
 /b-l475e-iot01a
+/b-l4s5i-iot01a
+/stm32h745i-disco
+
+# Gate build directories
+build_pc_linux
```

Comment:
is there reason why we have absolute path for above and relative for these?

Reply:
ah nope bad copy paste